src/stylesheets/common/_typography.scss

```diff
@@ -60,3 +60,81 @@ h5 {
   line-height: 1.4;
   color: $mz-mediumgray;
 }
+
+
+.body-text {
+  font-family: 'Open Sans', sans-serif;
+  font-weight: 400;
+  font-size: 18px;
+  line-height: 28px;
+  color: $mz-darkgray;
+}
+
+.link {
+  font-family: 'Open Sans', sans-serif;
+  font-weight: 600;
+  font-size: 18px;
+  line-height: 28px;
+  color: $mz-blue;
+
+  &:hover {
+    color: $mz-darkblue;
+  }
+}
+
+.caption {
+    font-size: 16px;
+    font-weight: 300;
+    font-style: italic;
+    color: $mz-mediumgray;
+}
+
+// Inline code
+code {
+  padding: 4px 8px;
+  margin: 0 2px;
+  font-size: 14px;
+  line-height: 1.6em;
+  color: $mz-mediumgray;
+  // Ensure inline code elements have sufficient
+  // contrast to differentiate between pure white
+  background-color: $mz-lightgray;
+}
+
+// Normal prose preformatted text
+pre {
```

Comment:
I wonder this can break documentation's style (documentation site is using only `pre` for code) , let me test this